src/Components/Common/Breadcrumbs.tsx

```diff
@@ -140,7 +140,12 @@ export default function Breadcrumbs(props: any) {
                         </div>
                       ) : (
                         <div>
-                          <span>{crumb.name}</span>
+                          <span className="hidden md:block">{crumb.name}</span>
+                          <span className="block md:hidden">
+                            {crumb.name.length > 13
+                              ? crumb.name.slice(0, 13) + "..."
+                              : crumb.name}
+                          </span>
```

Comment:
@Pranshu1902 Use  [`truncate`](https://tailwindcss.com/docs/text-overflow#truncate) class instead

Reply:
I tried with `truncate` but it still wasn't fixing the issue